package.json

```diff
@@ -26,7 +26,8 @@
     "grunt-mocha-test": "^0.12.7",
     "mocha": "^2.1.0",
     "requirejs": "^2.1.16",
-    "sinon": "^1.12.2"
+    "sinon": "^1.12.2",
+    "underscore": "^1.8.3"
```

Comment:
this dep will no longer be needed, please create a relevant requirejs dep instead